test/node-api/test_threadsafe_function/test.js

```diff
@@ -43,7 +43,7 @@ function testWithJSMarshaller({
     binding[threadStarter](function testCallback(value) {
       array.push(value);
       if (array.length === quitAfter) {
-        setImmediate(() => {
+        process.nextTick(() => {
```

Comment:
Unfortunately, the iteration count is too low to trigger the starvation prevention above. I guess I should add a test that makes more iterations, but the whole binding is about doing calling `usleep()` during each iteration, so I think this is going to be slow :joy:

Any ideas on how to test it?